Dockerfile

```diff
@@ -23,17 +23,19 @@ RUN ./gradlew assemble
 
 
 # ---
-FROM eclipse-temurin:17.0.6_10-jre-alpine AS final
+FROM eclipse-temurin:17.0.6_10-jre-focal AS final
```

Comment:
_not a blocker_
Does this need `--platform=${BUILDPLATFORM:-linux/amd64}` too?

Reply:
Yes not required, changing it now.